cdm/src/main/java/ucar/nc2/ft2/coverage/simpgeometry/SimpleGeometryCoverage.java

```diff
@@ -26,39 +27,100 @@
  * Immutable after setCoordSys() is called.
  *
  * @author Katie
- * @author Carron
+ * @author Caron
  * @since 8/13/2018
  */
 // @Immutable
-public class SimpleGeometryCoverage implements VariableSimpleIF, IsMissingEvaluator { //add interface with geometry info? //subsetting data will be diffferent?? is it subset now

```

Comment:
Do we still want to remove coverage?

Reply:
The Coverage name? Could possibly make refactoring that another class (most likely, such a refactoring will involve moving the simpgeometry coverage subpackage out of the Coverage package.

Reply:
MAP-94 is the resulting ticket.